src/app/Agent/AgentProbeTemplates.tsx

```diff
@@ -330,7 +330,10 @@ export const AgentProbeTemplates = () => {
     return (
       <ErrorView
         title={'Error retrieving probe templates'}
-        message={errorMessage}
+        message={
+          'This is commonly caused by the agent not being loaded/active, check that the target was started with the agent (-javaagent:/path/to/agent.jar) ' +
```

Comment:
Would start a new sentence with `This is commonly caused by the agent not being loaded/active. Check that...`

Reply:
Whoops i'm not meaning to approve.